ibm_catalog.json

```diff
@@ -138,6 +139,9 @@
             {
               "key": "prefix"
             },
+            {
+              "key":"cbr_rules"
+            },
```

Comment:
In this json file, ordering of variables matters. So we should place it logically. I dont think, it fits well before the instance name. I suggest to move down after keys.

Reply:
done https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/pull/594/commits/67a2aa1c5119d9bbaed014260a89ecfed7c12f95